src/query/shorthand.ts

```diff
@@ -91,6 +100,54 @@ export function spec(specQ: SpecQuery,
   return parts.join('|');
 }
 
+let getFilterExpression = filter.getFilterExpression;
```

Comment:
I'm considering changing `filter` to filterNameSpace so I can change the name of the method below.